### PR TITLE
Bug 1196624 - Need official and unofficial 336 pixel Firefox branding…

### PR DESCRIPTION
**Pull request by danhuang1202**
… icon for Firefox OS TV, r=rexboy

*add official/browser_336.png and unofficial/browser_336.png
*replace tv browse app branding icon from browser_284.png to browser_336.png

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=3fea714908ac85ffc110b32a9409baa502585a47

File: tv_apps/browser/index.html [hunk 1]
```diff
@@ -15,6 +15,9 @@
     <link rel="stylesheet" type="text/css" href="style/search_result.css">
     <link rel="stylesheet" type="text/css" href="style/settings.css">
 
+    <!-- Resources -->
+    <link rel="resource" type="/image/png" href="shared/resources/branding/browser_336.png">
```
Comment (rexboy7) [inline tv_apps/browser/index.html]: `type="image/png"`

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=1cd05bdd5f99b2871614f92f5e53b2b13712175b

**Comment by danhuang1202** [comment]
update type="image/png"

**Pull request merged**